suite-common/wallet-utils/src/transactionUtils.ts

```diff
@@ -46,30 +46,47 @@ export const parseDateKey = (key: string) => {
     return d;
 };
 
+/* Convert date to string in YYYY-MM format */
+export const getMonthKey = (d: Date) => `${d.getFullYear()}-${d.getMonth() + 1}`;
```

Comment:
Don't need to be exported, used just here. Place it right before `groupTransactionsByDate` because it's relavant only for this function.

Also please rename this to something like `(generate/prepare)MonthKey`

Reply:
fixed [30b5f1a](https://github.com/trezor/trezor-suite/pull/7392/commits/30b5f1a27811ed560da7091928ce739d439d0ca9)